inc/policybase.class.php

```diff
@@ -240,9 +240,12 @@ public function post_unapply($value, $itemtype, $itemId, PluginFlyvemdmNotifiabl
     * @return string
     */
    public function showValueInput($value = '', $itemType = '', $itemId = 0) {
-      $html = '<input name="value" value="' . $value . '" >';
 
-      return $html;
+      $data['typeTmpl'] = 'PluginFlyvemdmPolicyBase';
+      $data['itemtype'] = $itemType;
+      $data['value'] = $value;
+      $twig = plugin_flyvemdm_getTemplateEngine();
+      return $twig->render('tmpl_policy_value.html.twig', ['data' => $data]);
```

Comment:
The extension .twig telle the file is a template. It is useless to prefix with tmpl_